src/libaudcore/tuple.h

```diff
@@ -105,6 +105,7 @@ class Tuple
         /* TODO: reorder these at next ABI break! */
         Description,   /* Track description */
         MusicBrainzID, /* MusicBrainz identifier */
+        Lyrics,        /* Lyrics from id3 tags. */
```

Comment:
Needs to be added at the end, after CatalogNum.

Reply:
Done